libass/ass_bitmap_engine.c

```diff
@@ -185,20 +200,15 @@ BitmapEngine ass_bitmap_engine_init(unsigned mask)
             BLUR_FUNCTION(expand_horz, 16, ssse3)
             PARAM_BLUR_FUNCTION(horz, 16, ssse3)
         }
-        return engine;
     }
 #elif ARCH_AARCH64
     if (flags & ASS_CPU_FLAG_ARM_NEON) {
         ALL_PROTOTYPES(16, neon)
-        ALL_FUNCTIONS(4, 16, neon)
-        return engine;
+        BASE_FUNCTIONS(4, 16, neon)
+        SHIFT_FUNCTIONS(neon)
```

Comment:
nit: this could use `ALL_FUNCTIONS` making it clear no assembly routines are missing atm

Reply:
I kinda prefer to be explicit about this until functions get added to the BASE set.

Reply:
i don’t feel strongly about this; feel free to ignore